### ISSUE TITLE
Keycloak fails to start with "docker-compose up"

### ISSUE THREAD

**Issue by kenmeacham** [SPYDERISK/system-modeller#15]
I noticed this first yesterday, when using the #10 branch. When you run "docker-compose up -d", keycloak fails to start up. If you then run "docker-compose ps", you get the following:

```
vagrant@ssm3:/vagrant/SPYDERISK/system-modeller$ docker-compose ps
           Name                        Command             State                                             Ports
------------------------------------------------------------------------------------------------------------------------------------------------------------
system-modeller_keycloak_1   /tmp/import/entrypoint.sh     Exit 1
system-modeller_mongo_1      docker-entrypoint.sh mongod   Up       27017/tcp
system-modeller_ssm_1        tail -f /dev/null             Up       0.0.0.0:3000->3000/tcp,:::3000->3000/tcp, 0.0.0.0:5005->5005/tcp,:::5005->5005/tcp,
                                                                    0.0.0.0:8081->8081/tcp,:::8081->8081/tcp
```

Checking the logs, you get the following:

```
vagrant@ssm3:/vagrant/SPYDERISK/system-modeller$ docker-compose logs keycloak
Attaching to system-modeller_keycloak_1
keycloak_1  | exec /tmp/import/entrypoint.sh: no such file or directory
```

I switched back to the dev branch which still had the problem.

**Comment by kenmeacham** [comment]
Initial investigation seems to indicate a recent change to the docker-compose.yml file in dev, as the problem does not occur in the #3 branch, for example.

**Comment by scp93ch** [comment]
I don't find any problem. I:

* checked out dev
* docker-compose down -v
* docker-compose up -d
* docker-compose logs -f keycloak   [all looks fine]
* docker-compose ps
```
           Name                        Command             State                               Ports
---------------------------------------------------------------------------------------------------------------------------------
system-modeller_keycloak_1   /tmp/import/entrypoint.sh     Up      0.0.0.0:8080->8080/tcp, 8443/tcp
system-modeller_mongo_1      docker-entrypoint.sh mongod   Up      27017/tcp
system-modeller_ssm_1        tail -f /dev/null             Up      0.0.0.0:3000->3000/tcp, 0.0.0.0:5005->5005/tcp,
                                                                   0.0.0.0:8081->8081/tcp
```

Perhaps you had some stale data in a volume?

Can you re-test please?

**Comment by kenmeacham** [comment]
@scp93ch - I had already tried the same commands, but I tried it all again just in case. Still had same problem..

I then created a completely new VM and fresh checkout of SSM. No difference.

I then suspected that entrypoint.sh was not recognised as a UNIX script. After checking the type of file, it appears that it had CLRF line endings. Running "dos2unix" on this file fixed the problem.

I have therefore made a patch to .gitattributes to ensure that *.sh files are checked out as UNIX.

Just submitted a pull request for review..
